DEVELOPMENT.md

```diff
@@ -27,9 +27,23 @@ npm install
 npm run setup
 ```
 
+### Auth0 tenant set up
+
+JunctionApp uses Auth0 for user authentication and authorization. To set up Auth0 for local development:
+
+1. Create an Auth0 account
+2. Follow [this](https://auth0.com/docs/deploy/deploy-cli-tool/create-and-configure-the-deploy-cli-application) tutorial to create a `auth0-deploy-cli-extension` in Auth0
+3. Add Auth0 Authorization extension to your tenant.
```

Comment:
It seems this and the next instruction is conflicting/not in sync with the instructions in Auth0 tutorial (2.).